libcfnet/client_code.c

```diff
@@ -574,11 +574,11 @@ static bool EncryptCopyRegularFileNet(const char *source, const char *dest, off_
 {
     int blocksize = 2048, n_read = 0, plainlen, more = true, finlen;
     int tosend, cipherlen = 0;
-    char *buf, in[CF_BUFSIZE], out[CF_BUFSIZE], workbuf[CF_BUFSIZE], cfchangedstr[265];
+    char *buf, in[CF_BUFSIZE], out[CF_BUFSIZE], workbuf[CF_BUFSIZE], cfchangedstr[sizeof(CF_CHANGEDSTR1 CF_CHANGEDSTR2)];
     unsigned char iv[32] =
         { 1, 2, 3, 4, 5, 6, 7, 8, 1, 2, 3, 4, 5, 6, 7, 8, 1, 2, 3, 4, 5, 6, 7, 8, 1, 2, 3, 4, 5, 6, 7, 8 };
 
-    snprintf(cfchangedstr, 255, "%s%s", CF_CHANGEDSTR1, CF_CHANGEDSTR2);
+    snprintf(cfchangedstr, sizeof(cfchangedstr), "%s%s", CF_CHANGEDSTR1, CF_CHANGEDSTR2);
```

Comment:
Pre-existing, but no need to use `snprintf` here. Maybe you could change it to assign `cfchangedstr` like this:
```C
const char *cfchangedstr = CF_CHANGEDSTR1 CF_CHANGEDSTR2;
```